### DISCUSSION TITLE
Which day and time would you be most likely to attend a regular (weekly) Life Itself Community Hangout online?

### DISCUSSION THREAD

**Discussion by laurenwigmore** [life-itself/community#1004] [Polls]
We want to create an online community hangouts space, for the Life Itself community to gather, get to know each other and have a chat around what is particularly alive for us that week.

Our initial idea is to host these weekly for 1 hour.

We would love to know if you have a preference on a day/time from the ones given below that you would commit to attending regularly.

You can only select one option, but if you want to add more, or thoughts, please do so in the comments!

**Comment by asimong** [comment]
I would enjoy this being followed by the research meeting, as I feel there could be a nice synergy

**Comment by rufuspollock** [comment]
I'd generally have a time that allows a bit of lingering so you don't want this straight before another meeting IMO (though on the same day would be fine). for that reason 2pm friday is a big tight with current research collective meeting at 3pm CET. Plus 2pm won't work that well for US TZ (especially west coast).

In medium term if these are popular we could alternative times each week with one at a time that works for e.g. US and one that works for Australia etc.

**Comment by laurenwigmore** [comment]
<img width="665" alt="image" src="https://github.com/life-itself/community/assets/132304704/3f32e3b8-391a-4ba6-b536-b2a6d7ff7f45">
Example of late afternoon times in multiple locations
